alphabeta/heuristic_static_values.pl

```diff
@@ -3,6 +3,18 @@
  * for calculating static values for a position
  * using a heuristic predicates
  * */
+:- consult('heuristic_calc.pl'). % import possible states impl.
 % TODO, check max and min values of the hueristic function to use as parameters in alpha beta
+?- assert(saved_value(0,0)). % dummy value for compiler.
 
-staticval(_Pos,_Val). % TODO: impl heuristic_static_values for start/middle and endgame.
\ No newline at end of file
+ % TODO: impl heuristic_static_values for start/middle and endgame.
+staticval(Pos, Val):-
+    % value us already in db
+    retract(saved_value(Pos, Val)), ! % remove from db
```

Comment:
why removing? you can just use 
'''
saved_value(Pos,Val).
'''
it will fail if it doesn't exist

Reply:
done